generated-project/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "test",
+  "version": "0.0.1-dev.0",
+  "license": "See license in LICENSE",
+  "engines": {
+    "node": "^16.0.0 || ^18.0.0",
+    "npm": "^7.0.0 || ^8.0.0 || ^9.0.0"
+  },
+  "ccExtensibility": {
+    "extends": "retail-react-app",
+    "overridesDir": "overrides"
+  },
+  "devDependencies": {
+    "retail-react-app": "3.0.0-dev"
+  },
+  "scripts": {
+    "analyze-build": "cross-env MOBIFY_ANALYZE=true npm run build",
+    "build": "npm run build-translations && pwa-kit-dev build",
+    "build-translations": "npm run extract-default-translations && npm run compile-translations",
+    "compile-translations": "formatjs compile-folder --ast translations translations/compiled",
+    "compile-translations:pseudo": "formatjs compile --ast translations/en-US.json --out-file app/translations/compiled/en-XB.json --pseudo-locale en-XB",
+    "extract-default-translations": "node ./node_modules/retail-react-app/scripts/build-translations.js",
```

Comment:
This command is currently different from baseTemplate which can be changed during generating process